daemon/exec.go

```diff
@@ -217,12 +216,17 @@ func (d *Daemon) ContainerExecStart(ctx context.Context, name string, stdin io.R
 		ec.StreamConfig.NewNopInputPipe()
 	}
 
-	p := &specs.Process{
-		Args:     append([]string{ec.Entrypoint}, ec.Args...),
-		Env:      ec.Env,
-		Terminal: ec.Tty,
-		Cwd:      ec.WorkingDir,
+	spec, err := d.createSpec(c)
+	if err != nil {
+		return errdefs.System(err)
```

Comment:
I know the other place we call `createSpec` is wrapping this error, unfortunately it's wrong and is being addressed in another PR. Can we not wrap here?